src/app/header/Header.tsx

```diff
@@ -1,24 +1,37 @@
-import React from 'react';
-// import StyledHeader from './Header.styles';
-
+import React from "react";
+import { Box, Flex, HStack, Spacer, Text } from "@chakra-ui/react";
+import { SearchBar } from "./components/SearchBar";
+import { UserButton } from "./components/UserButton";
+import { TermButtons } from "./components/TermButtons";
 
 export interface HeaderProps {
   /**
-   * Content
-   * Subject to change
+   * setTerm: changes the state of the term selected with the buttons
    */
-  content?: string;
+  setTerm: React.Dispatch<React.SetStateAction<string>>;
 }
 
 /**
  * Primary UI component for content
  */
-export const Header: React.FC<HeaderProps> = ({
-  content
-}) => {
+export const Header: React.FC<HeaderProps> = ({ setTerm }) => {
```

Comment:
@zthh626 and I were dicussing and after some Googling around we came to the conclusion that the normal `function ({}: Props): JSX.Element` is the way to go.

Reply:
yeah I prefer that too